TestingLowerBounds/FDiv/CondFDiv.lean

```diff
@@ -263,6 +263,20 @@ lemma condFDiv_eq' (hf_ae : ∀ᵐ a ∂μ, fDiv f (κ a) (η a) ≠ ⊤)
     condFDiv f κ η μ = ((μ[fun x ↦ (fDiv f (κ x) (η x)).toReal] : ℝ) : EReal) :=
   if_pos ⟨hf_ae, hf⟩
 
+lemma condFDiv_nonneg [IsMarkovKernel κ] [IsMarkovKernel η]
+    (hf_cvx : ConvexOn ℝ (Set.Ici 0) f) (hf_cont : ContinuousOn f (Set.Ici 0))
+    (hf_one : f 1 = 0) : 0 ≤ condFDiv f κ η μ := by
```

Comment:
Is there a blueprint entry for this? If yes, can you link it to this lemma and if not, add one?